doc/dataframe.md

```diff
@@ -154,17 +154,16 @@ df.write
   .save()
 ```
 
-
 ### Persistence model
 
-By default DataFrames are persisted as Redis Hashes. It allows to write data with Spark and query from non-Spark environment.
+By default DataFrames are persisted as Redis Hashes. It allows for data to be written with Spark and queried from a non-Spark environment.
 It also enables projection query optimization when only a small subset of columns are selected. On the other hand, there is currently 
-a limitation with Hash model - it doesn't support nested DataFrame schema. One option to overcome it is making your DataFrame schema flat.
-If it is not possible due to some constraints, you may consider using Binary persistence model.
+a limitation with the Hash model - it doesn't support nested DataFrame schemas. One option to overcome this is to your DataFrame schema flat.
```

Comment:
`"One option to overcome this is to your DataFrame schema flat." `
should it be
"One option to overcome this is to make your DataFrame schema flat." ?

Reply:
Yep, fixed.